lib/index.d.ts

```diff
@@ -37,23 +37,6 @@ export type ServiceClientOptions = {
     timeout?: number;
 };
 
-export interface ServiceRequest {
```

Comment:
This library has no need for any specificity in this type, used for the context option, which is passed through to plugins without being accessed by this library